grapher/core/Grapher.tsx

```diff
@@ -520,9 +510,23 @@ export class Grapher
     }
 
     @computed
-    private get tableAfterPopulationAndActiveChartAndTimelineFilters() {
+    get tableAfterAuthorTimelineAndActiveChartTransformAndPopulationFilter() {
```

Comment:
I really like these methods names as a stopgap until we discover what the pattern is that we should be using. But then again, I think of all the data science scripts and really not sure if there is a better pattern. Maybe this will be the long term solution too.

Reply:
Part of the issue is also that it's better performance to have computations chopped up in as many steps as possible, with computed props for each, so that mobx can optimize and only update when an update is necessary.